configs/bsc/wsteth-mainnet.yml

```diff
@@ -0,0 +1,666 @@
+        getOutboundQueuedTransfer:
+          - args: [ 0 ]
+            result: [ *ZERO_BYTES32, *ZERO_BYTES32, 0, 0, 0, *ZERO_ADDRESS, "0x" ]
+        getPeer:
+          - args: [ *ETH_CHAIN_ID ]
+            result: [ *ZERO_BYTES32, 0 ]
+          - args: [ *BSC_CHAIN_ID ]
+            result: [ *ZERO_BYTES32, 0 ]
+        getThreshold: 0
+        getTransceiverInfo: []
+        getTransceivers: []
+        isMessageApproved:
+          - args: [ *ZERO_BYTES32 ]
+            result: false
+        isMessageExecuted:
+          - args: [ *ZERO_BYTES32 ]
+            result: false
+        isPaused: false # false
+        messageAttestations:
+         - args: [ *ZERO_BYTES32 ]
+           result: 0
+        mode: *LOCKING_MODE
+        nextMessageSequence: 0
+        owner: *ZERO_ADDRESS
+        pauser: *ZERO_ADDRESS
+        quoteDeliveryPrice:
+          - args: [ *BSC_CHAIN_ID, 0x ]
+            result: [[], 0]
+            mustRevert: true
+        rateLimitDuration: *RATE_LIMIT_DURATION # 86400
+        token: *L1_WSTETH # 0x7f39C581F595B53c5cb19bD0b3f8dA6c935E2Ca0
+        tokenDecimals: *DECIMALS # 18
+        transceiverAttestedToMessage:
+          - args: [ *ZERO_BYTES32, 0 ]
+            result: false
+          - args: [ *ZERO_BYTES32, 1 ]
+            result: false
+    l1WormholeTransceiver:
+      name: WormholeTransceiver
+      address: *l1WormholeTransceiver
+      proxyName: ERC1967Proxy
+      implementation: *l1WormholeTransceiverImpl
+      proxyChecks: {}
+      checks:
+        WORMHOLE_TRANSCEIVER_VERSION: *NTT_MANAGER_VERSION # 1.1.0
+        consistencyLevel: 1
+        encodeWormholeTransceiverInstruction:
+          - args: [ ]
+            result: [ ]
+            mustRevert: true
+        gasLimit: 500000
+        getMigratesImmutables: false
+        getNttManagerOwner: *L1_NEW_INTERIM_MSIG
+        getNttManagerToken: *L1_WSTETH # 0x7f39C581F595B53c5cb19bD0b3f8dA6c935E2Ca0
+        getTransceiverType: "wormhole"
+        getWormholePeer:
+          - args: [ *ETH_CHAIN_ID ]
+            result: *ZERO_BYTES32
+          - args: [ *BSC_CHAIN_ID ]
+            result: *L2_WORMHOLE_TRANSCEIVER_BYTES32_ADDR
+        isPaused: false
+        isSpecialRelayingEnabled:
+          - args: [ *ETH_CHAIN_ID ]
+            result: false
+          - args: [ *BSC_CHAIN_ID ]
+            result: false
+        isVAAConsumed:
+          - args: [ *ZERO_BYTES32 ]
+            result: false
+        isWormholeEvmChain:
+          - args: [ *ETH_CHAIN_ID ]
+            result: false # how come ???
+          - args: [ *BSC_CHAIN_ID ]
+            result: true
+        isWormholeRelayingEnabled:
+          - args: [ *ETH_CHAIN_ID ]
+            result: false
+          - args: [ *BSC_CHAIN_ID ]
+            result: true
+        nttManager: *l1NttManager
+        nttManagerToken: *L1_WSTETH
+        owner: *L1_NEW_INTERIM_MSIG
+        parseWormholeTransceiverInstruction:
+          - args: [ 0x ]
```

Comment:
It is more correct to write `"0x"` because otherwise yaml parses this as hex number to zero number.

In general I think state-mate must verify that view bytes type arg accepts string not number. Added it to personal TODO list

Reply:
yep, ty!